src/store/data.json

```diff
@@ -0,0 +1,133 @@
+{
```

Comment:
even tough you'll fetch this data from api later on, please add proper questions and remove this dummy questions and answers

Reply:
have removed this file as i've integrated api locally, will create a PR soon